src/error.rs

```diff
@@ -238,6 +238,14 @@ impl Error {
         matches!(self.inner.kind, Kind::User(User::BodyWriteAborted))
     }
 
+    /// Returns true if the error was caused while calling `AsyncWrite::shutdown()`.
+    pub fn is_shutdown(&self) -> bool {
+        #[cfg(all(feature = "http1", any(feature = "client", feature = "server")))]
+        matches!(self.inner.kind, Kind::Shutdown);
```

Comment:
You'll need to remove the semicolon

```suggestion
        matches!(self.inner.kind, Kind::Shutdown)
```

Reply:
oops - yep